discovery-provider/plugins/verified-uploads/src/db.js

```diff
@@ -0,0 +1,41 @@
+import Knex from "knex";
+import dotenv from "dotenv";
+const { knex } = Knex;
+
+const DB = async (url) => {
+  const pg = knex({
+    client: "pg",
+    connection: url,
+    pool: { min: 2, max: 10 },
+  });
+  console.log(`opening connection to ${url}`);
+  return pg;
+};
+
+// GLOBAL db handles
+dotenv.config();
+const { DISCOVERY_DB_URL } = process.env;
+export const dp_db = await DB(DISCOVERY_DB_URL).catch(console.error);
+
+const shouldToggleOff = (topic) => {
+  const { TOGGLE_OFF } = process.env;
+  const toggledOffTopics = TOGGLE_OFF.split(",");
+  const shouldToggle = toggledOffTopics.includes(topic) ? true : false;
+  if (shouldToggle) {
+    console.warn(`toggling off listener for topic '${topic}'`);
```

Comment:
for my learning what is a topic referencing?

Reply:
This is the topic for the pg_notify call. So it would be something like "tracks" or "users".